conf/machine/include/imx-base.inc

```diff
@@ -281,6 +281,15 @@ MACHINE_GSTREAMER_1_0_PLUGIN_mx7d   ?= "gstreamer1.0-plugins-imx-meta"
 MACHINE_GSTREAMER_1_0_PLUGIN_mx8mm  ?= "imx-gst1.0-plugin"
 MACHINE_GSTREAMER_1_0_PLUGIN_mx8mn  ?= "imx-gst1.0-plugin"
 MACHINE_GSTREAMER_1_0_PLUGIN_mx8mp  ?= "imx-gst1.0-plugin"
+MACHINE_GSTREAMER_1_0_PLUGIN_mx8mq  ?= "imx-gst1.0-plugin"
+MACHINE_GSTREAMER_1_0_PLUGIN_mx8qm  ?= "imx-gst1.0-plugin"
+MACHINE_GSTREAMER_1_0_PLUGIN_mx8qxp ?= "imx-gst1.0-plugin"
+
+PREFERRED_VERSION_gstreamer1.0_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-base_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-good_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-bad_mx8 ?= "1.16.imx"
```

Comment:
`MACHINE_GSTREAMER_1_0_PLUGIN` and `PREFERRED_VERSION_gstreamer1.0` are also defined in [meta-freescale-distro/conf/distro/include/fsl-base.inc](https://github.com/Freescale/meta-freescale-distro/blob/063a082a241772b7675daa258822954554160b65/conf/distro/include/fsl-base.inc#L60)

Since I do not see a use case where someone would pull the _meta-freescale-distro_ without pulling _meta-freescale_ - would it make sense to move all defines in here and clean them up in the disto layer? They seem to be scattered now between 2 layers, and defined differently for mx6/mx7 machines.

Reply:
Those serve different purposes:

- `imx-base.inc` aims to provide the BSP default. For all SoCs but the i.MX8 we default to `gstreamer-1.0-imx` which supports NXP BSP and has been developed by the community. It does not yet offer i.MX8 support and @dv1 is working to solve this gap.

- `fsl-*` distros are to provide an NXP BSP very close to their GA releases but using up to date underlying Operating System.

In future the i.MX8 SoCs are also going to use the `gstreamer-1.0-imx` provider and then the `imx-gst1.0-plugin` will be used only on `fsl-*` distros.

Reply:
OK, got it! This actually serves a "reverse logic" where if the _-distro_ is not pulled in - it defaults to what is inside this conf files, otherwise - it opts-in to choose a proper config based on the BSP flavor.

Good to know, thanks a lot!

Then I guess proper `MACHINE_GSTREAMER_1_0_PLUGIN` defines for `mx8mm/mx8mn/mx8mp` are missing in the _-distro_ layer...

Reply:
@zandrey can you take care of the distro?

Reply:
> @zandrey can you take care of the distro?

Sorry I didn't give and update here before, PR [41](https://github.com/Freescale/meta-freescale-distro/pull/41) submitted against _meta-freescale-distro_.

Reply:
I merged it already.